### ISSUE TITLE
Occasional SNS Publish errors

### ISSUE THREAD

**Issue by hwine** [mozilla-services/fx-sig-verify#26]
``analyze_logfiles`` found this a couple of times:
```
An error occurred (InvalidParameter) when calling the Publish operation: Invalid parameter: Subject: InvalidParameterException
Traceback (most recent call last):
  File "/var/task/fleece/xray.py", line 354, in wrapper
    metadata_extractor=metadata_extractor,
  File "/var/task/fleece/xray.py", line 239, in generic_xray_wrapper
    return wrapped(*args, **kwargs)
  File "/var/task/fx_sig_verify/validate_moz_signature.py", line 526, in lambda_handler
    artifact.report_validity()
  File "/var/task/fx_sig_verify/validate_moz_signature.py", line 336, in report_validity
    self.send_sns(message)
  File "/var/task/fleece/xray.py", line 354, in wrapper
    metadata_extractor=metadata_extractor,
  File "/var/task/fleece/xray.py", line 239, in generic_xray_wrapper
    return wrapped(*args, **kwargs)
  File "/var/task/fx_sig_verify/validate_moz_signature.py", line 418, in send_sns
    TopicArn=topic_arn)
  File "/var/task/botocore/client.py", line 310, in _api_call
    return self._make_api_call(operation_name, kwargs)
  File "/var/task/fleece/xray.py", line 424, in xray_botocore_api_call
    error_handling_type=ERROR_HANDLING_BOTOCORE,
  File "/var/task/fleece/xray.py", line 239, in generic_xray_wrapper
    return wrapped(*args, **kwargs)
  File "/var/task/botocore/client.py", line 599, in _make_api_call
    raise error_class(parsed_response, operation_name)
InvalidParameterException: An error occurred (InvalidParameter) when calling the Publish operation: Invalid parameter: Subject
END RequestId: 347a2208-8914-11e7-90fb-6fd08f57a948
```
From prior experience, this means the *value* of the ``Subject`` parameter is invalid. The odd thing is, the same request id completes correctly in both instances.

**Comment by hwine** [comment]
This is showing up "a lot" now as we examine all files (not just exe's [v0.4.0](https://github.com/mozilla-services/fx-sig-verify/milestone/3?closed=1)), but root cause is at hand:
 - added additional debugging code in 20dc5ab4014d5f440b453910c4f2cb24a922de21
 - runs in staging with v0.4.0 showed it is long SNS subject strings that are the problem
   - max subject length is 100 chars
   - was using full s3 URL, which often is closer to 150 characters

**Comment by hwine** [comment]
Fixed in e3c002c374c21b45e36ea68568de9c94a9cbfc51
